md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ProcessLogAPI.java

```diff
@@ -54,11 +51,8 @@
     public
     @ResponseBody
     RestWrapper list(@RequestParam(value = "page", defaultValue = "0") int startPage, @RequestParam(value = "size", defaultValue = "10") int pageSize, @RequestParam(value = "pid", defaultValue = "0") Integer pid, Principal principal) {
-        // SqlSession s = null;
         RestWrapper restWrapper = null;
         try {
-            // SqlSessionFactory sqlSessionFactory = getSqlSessionFactory(null);
-            // s = sqlSessionFactory.openSession();
             ProcessLogInfo processLogInfo = new ProcessLogInfo();
             if (pid == 0) {
                 pid = null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Introduce a new variable instead of reusing the parameter "pid". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1226)